test/test_metalearning/pyMetaLearn/test_metalearner.py

```diff
@@ -35,7 +35,7 @@ def tearDown(self):
 
     def test_metalearning_suggest_all(self):
         ret = self.meta_optimizer.metalearning_suggest_all()
-        self.assertEqual(124, len(ret))
+        self.assertEqual(120, len(ret))
```

Comment:
The reason this number changed is because the LDA configuration(s) can no longer be loaded because you removed one hyperparameter. If you want to you can go through all `configurations.csv` files and remove it yourself, or I'll do so once this is merged.